kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/util/PppAuthSecrets.java

```diff
@@ -36,12 +36,9 @@
 public class PppAuthSecrets {
 
     /*
-     * #Secrets for authentication using PAP/CHAP
-     * #client server secret IP addresses Provider
-     * ISP@CINGULARGPRS.COM * CINGULAR1 * #att
-     * mobileweb * password * #o2
-     * user * pass * #orange
-     * web * web * #vodaphone
+     * #Secrets for authentication using PAP/CHAP #client server secret IP addresses
```

Comment:
Hi, thanks for the PR.
Are those changes to the comments needed?

Reply:
Hi , without those changes , the build was throwing Tab-Space indentation problem for the whole file line by line. So , I had to apply Tab to Space conversion for the full file .

No change in content was done for the documentations.